dbusmock/templates/accounts_service.py

```diff
@@ -0,0 +1,392 @@
+        properties = mock.GetAll(interface)
+    elif isinstance(properties, str):
+        properties = [properties]
+
+    if isinstance(properties, (list, set)):
+        properties = {p: mock.Get(interface, p) for p in properties}
+    elif isinstance(properties, dict):
+        pass
+
+    mock.EmitSignal(dbus.PROPERTIES_IFACE, 'PropertiesChanged', 'sa{sv}as', (
+        interface, properties, []))
+
+
+@dbus.service.method(MOCK_IFACE, in_signature='xsa{sv}', out_signature='o')
+def AddUser(self, uid, username, overrides=None):
+    '''Add user via uid and username and optionally overriding properties
+
+    Returns the new object path.
+   '''
+    path = get_user_path(uid)
+    default_props = {
+        'Uid': dbus.UInt64(uid),
+        'UserName': username,
+        'RealName': username[0].upper() + username[1:] + ' Fake',
+        'AccountType': dbus.Int32(1),
+        'AutomaticLogin': False,
+        'BackgroundFile': '',
+        'Email': '{}@python-dbusmock.org'.format(username),
+        'FormatsLocale': 'C',
+        'HomeDirectory': '/tmp/dbusmock-home/{}'.format(username),
+        'IconFile': '',
+        'InputSources': dbus.Array([], signature='a{ss}'),
+        'Language': 'C',
+        'LocalAccount': True,
+        'Location': '',
+        'Locked': False,
+        'LoginFrequency': dbus.UInt64(0),
+        'LoginHistory': dbus.Array([], signature='(xxa{sv})'),
+        'LoginTime': dbus.Int64(0),
+        'PasswordHint': 'Remember it, come on!',
+        'PasswordMode': 0,
+        'Session': 'mock-session',
+        'SessionType': 'wayland',
+        'Shell': '/bin/zsh',
+        'SystemAccount': False,
+        'XHasMessages': False,
+        'XKeyboardLayouts': dbus.Array([], signature='s'),
+        'XSession': 'mock-xsession',
+    }
+    default_props.update(overrides if overrides else {})
+    self.AddObject(path, USER_IFACE, default_props, [
+        ('SetRealName', 's', '', SetRealName),
+        ('SetEmail', 's', '', SetEmail),
+        ('SetLanguage', 's', '', SetLanguage),
+        ('SetXSession', 's', '', SetXSession),
+        ('SetSession', 's', '', SetSession),
+        ('SetSessionType', 's', '', SetSessionType),
+        ('SetLocation', 's', '', SetLocation),
+        ('SetHomeDirectory', 's', '', SetHomeDirectory),
+        ('SetShell', 's', '', SetShell),
+        ('SetIconFile', 's', '', SetIconFile),
+        ('SetLocked', 'b', '', SetLocked),
+        ('SetAccountType', 'i', '', SetAccountType),
+        ('SetPasswordMode', 'i', '', SetPasswordMode),
+        ('SetPasswordHint', 's', '', SetPasswordHint),
+        ('SetPassword', 'ss', '', SetPassword),
+        ('SetAutomaticLogin', 'b', '', SetAutomaticLogin),
+        ('GetPasswordExpirationPolicy', '',
+         'xxxxxx', GetPasswordExpirationPolicy),
+    ])
+
+    had_users = len(self.mock_users) != 0
+    had_multiple_users = len(self.mock_users) > 1
+    user = mockobject.objects[path]
+    user.password = 'Pa$$wo0rd'
```

Comment:
Please define this as a `DEFAULT_PASSWORD` constant at the top of the file, so that it's easier to spot.